requirements.txt

```diff
@@ -10,6 +10,7 @@ Jinja2==2.8
 MarkupSafe==0.23
 py==1.4.31
 Pygments==2.1.1
+PySocks==1.5.6
```

Comment:
https://github.com/kennethreitz/requests/pull/3185#issuecomment-218884018

The new functional test tries to connect to a (fake) socks proxy, which requires PySocks.